cylc/flow/scheduler.py

```diff
@@ -1733,6 +1734,10 @@ async def main_loop(self) -> None:
                 if all(itask.is_ready_to_run()):
                     self.pool.queue_task(itask)
 
+            if self.xtrigger_mgr.sequential_spawn_next:
+                # Spawn parentless tasks with sequentially checked xtrigger(s).
```

Comment:
```suggestion
```
(Not a biggie, but with such verbose attribute names that comment is kinda redundant).